JSON/draft-07/schema_apidef_lean.json

```
@@ -177,53 +178,26 @@
                         }
                     ],
                     "required": [
-                        "Default"
                     ],
                     "properties": {
-                        "Default": {
-                            "$id": "#/properties/version_data/properties/versions/properties/Default",
-                            "type": "object",
-                            "title": "The Default schema",
-                            "description": "An explanation about the purpose of this instance.",
-                            "default": {},
-                            "examples": [
-                                {
-                                    "name": "Default"
-                                }
-                            ],
-                            "required": [
-                                "name"
-                            ],
-                            "properties": {
-                                "name": {
-                                    "$id": "#/properties/version_data/properties/versions/properties/Default/properties/name",
-                                    "type": "string",
-                                    "title": "The name schema",
-                                    "description": "An explanation about the purpose of this instance.",
-                                    "default": "",
-                                    "examples": [
-                                        "Default"
-                                    ]
-                                }
-                            },
-                            "additionalProperties": true
-                        }
                     },
                     "additionalProperties": true
                 },
                 "not_versioned": {
                     "$id": "#/properties/version_data/properties/not_versioned",
                     "type": "boolean",
                     "title": "The not_versioned schema",
-                    "description": "An explanation about the purpose of this instance.",
+                    "description": "Define if the api has versioning turned on. True by default, i.e. not versioned.",
```

Is version_data.not_versioned True by default? As far as i understand from this documentation https://tyk.io/docs/tyk-apis/tyk-gateway-api/api-definition-objects/versioning-endpoint/#versioning-in-the-url-path, the default value of  `version_data.not_versioned` is False.

> version_data.not_versioned: If set to true Tyk will skip version checking, you can still block or allow information to your API by specifying a Default version within the versions map.

```suggestion
                    "description": "Define if the api has versioning turned on. False by default, i.e. not versioned.",
```

---

@buraksekili False for `not_versioned` means it is versioned